src/main/groovy/com/smartbear/swagger/SwaggerUtils.groovy

```diff
@@ -21,13 +22,14 @@ class SwaggerUtils {
      *
      * @param url
      * @param project
+     * @param defaulMediaType
      * @return the corresponding SwaggerImporter based on the described "algorithm"
      */
 
-    static SwaggerImporter createSwaggerImporter(String url, WsdlProject project) {
+    static SwaggerImporter createSwaggerImporter(String url, WsdlProject project, String defaulMediaType) {
```

Comment:
please keep the old constructor that passes on to the new one with the default value from SwaggerUtils - I know we have others that use these classes and I don't want their code to break.